packages/tools/client-debugger/client-debugger-chrome-extension/package.json

```diff
@@ -41,7 +41,7 @@
 		"@fluentui/react": "^8.96.1",
 		"@fluentui/react-hooks": "^8.6.12",
 		"@fluentui/react-icons-mdl2": "^1.3.22",
-		"@fluentui/react-tabs": "^9.1.6",
+		"@fluentui/react-tabs": "^9.3.6",
```

Comment:
I think the new dep here is not necessary? debugger-client-view will have it and do whatever is needed with it in its exported components, right?

Reply:
Yeah, I suspect none of the FluentUI packages are needed here. Might be better to clean up in a separate PR though? Either way works for me.